examples/tv-casting-app/darwin/TvCasting/MatterBridge/CastingServerBridge.mm

```diff
@@ -48,15 +48,72 @@ - (instancetype)init
             return nil;
         }
 
+        // init app Server
+        static chip::CommonCaseDeviceServerInitParams initParams;
+        err = initParams.InitializeStaticResourcesBeforeServerInit();
+        if (err != CHIP_NO_ERROR) {
+            ChipLogError(AppServer, "InitializeStaticResourcesBeforeServerInit failed: %s", ErrorStr(err));
+            return nil;
+        }
+        err = chip::Server::GetInstance().Init(initParams);
+        if (err != CHIP_NO_ERROR) {
+            ChipLogError(AppServer, "chip::Server init failed: %s", ErrorStr(err));
+            return nil;
+        }
+
+        chip::DeviceLayer::PlatformMgrImpl().StartEventLoopTask();
+
         CastingServer::GetInstance()->Init();
     }
     return self;
 }
 
-// TBD: placeholder will be replaced with true CastingServer functions
-- (int)add:(int)a secondNum:(int)b
+- (bool)discoverCommissioners
 {
-    return a + b;
+    ChipLogProgress(AppServer, "CastingServerBridge().discoverCommissioners() called");
+    CHIP_ERROR err = CastingServer::GetInstance()->DiscoverCommissioners();
```

Comment:
Which queue is this running on?  As in, are there going to be thread races here?

Reply:
Updated the code, so this network call and the ones below are made asynchronously on the chipWorkQueue.